partner_bank_mail_thread/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner Bank Mail Thread',
+    'summary': """
+        Enable a chatter on the partner bank accounts which will track
+        updates done on a partner bank account""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/partner-contact',
+    'depends': [
+        'base',
```

Comment:
No need for this line. Only need to depend on base if there are no other modules.